src/mpyl/steps/deploy/k8s/resources/treafik.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
this file name also has a typo

Reply:
Where?

Reply:
Ah, thanks! What an impossible name 😱

Reply:
5291fb8d79381113b6acd2fcf7599512eff07e86